source/simulator/nullspace.cc

```diff
@@ -206,6 +454,22 @@ namespace aspect
   }
 
 
+  template <int dim>
+  void Simulator<dim>::setup_nullspace_constraints(AffineConstraints<double> &constraints,
+                                                   const DoFHandler<dim> &dof_handler,
+                                                   const int level)
```

Comment:
same thing here with the level variable, assuming you are again treating `-1` as special.